python/tvm/driver/tvmc/model.py

```diff
@@ -280,7 +284,10 @@ def export_package(
                 executor_factory, package_path, cross, cross_options, output_format
             )
         elif output_format == "mlf":
-            package_path = export_model_library_format(executor_factory, package_path)
+            if export_model_library_format is not None:
```

Comment:
I would simplify this as the following, like in other parts of the code:

```
-            if export_model_library_format is not None:
+            if export_model_library_format:
```

Reply:
fixed